src/main/resources/application.yaml

```diff
@@ -39,7 +39,7 @@ camunda:
   url: ${CAMUNDA_URL:http://camunda-local-bpm/engine-rest}
 
 task-configuration:
-  url: ${TASK_CONFIGURATION_SERVICE_URL:http://localhost:8091}
+  url: ${TASK_CONFIGURATION_SERVICE_URL:http://localhost:8087}
```

Comment:
TASK_MANAGEMENT_SERVICE_URL is already injected we can get rid of this property and use 
`task-management.url` instead